bin/ts.js

```diff
@@ -0,0 +1,25 @@
+    var fs = require('fs-extra');
+    var path = require('path');
+    var typedoc = require('typedoc');
+    var express = require('express');
+
+    module.exports = function ts(outputPath, previewMode){
+        
+        var filePaths = JSON.parse(fs.readFileSync("ts-docs.json")).content;
+
+        // Setup our TypeDoc app
+        var app = new typedoc.Application({
+            mode:   'Modules',
+            logger: 'console',
+            target: 'ES6',
+            module: 'CommonJS'
+        });
+        // Generate the docs
+        app.generateDocs(filePaths, outputPath);
+
+        if(previewMode){
+            var app = express();
+            app.use('/', express.static(process.cwd() + '/' + outputPath));
+            app.listen(3000);
+        }
+    }
```

Comment:
remove extra indentation in whole file and use 2 space indentation in entire file